oauth2_http/java/com/google/auth/oauth2/ExternalAccountAuthorizedUserCredentials.java

```diff
@@ -75,12 +77,19 @@
  * }
  * </pre>
  */
-public class ExternalAccountAuthorizedUserCredentials extends GoogleCredentials {
+public class ExternalAccountAuthorizedUserCredentials extends GoogleCredentials
+    implements TrustBoundaryProvider {
 
   private static final String PARSE_ERROR_PREFIX = "Error parsing token refresh response. ";
 
   private static final long serialVersionUID = -2181779590486283287L;
 
+  private static final String WORKFORCE_POOL_URL_FORMAT =
+      "https://iamcredentials.googleapis.com/v1/locations/global/workforcePools/%s/allowedLocations";
```

Comment:
Is using the GDU (googleapis.com) here always fine? Or should this be `https://iamcredentials.%s` and accept a universe domain?